docs/get-started/cost-billing/pricing-for-bare-metal-machines.md

```diff
@@ -0,0 +1,69 @@
+# Pricing for Bare Metal (BM) Machines on NERC
+
+## Bare Metal Machines
+
+!!! warning "Very Important Disclosures Regarding Pricing"
+
+    All prices listed on this page are **introductory** and subject to change
+    without notice. Please stay informed by reviewing this page regularly or
+    contacting us by emailing us at [help@esi.mghpcc.org](mailto:help@esi.mghpcc.org?subject=New%20User%20Query)
+    or, by submitting a new ticket at [the NERC's Support Ticketing System](https://mghpcc.supportsystem.com/open.php)
+    by selecting "**ESI Inquiry**" as the *Help Topic* option as shown below:
+
+    ![ESI Inquiry Help Topic](images/esi-help-topic.png)
+
+Bare metal instances pricing is currently available as a limited offering.
+
+## Bare Metal Pricing Calculations
+
+| Name             | vGPU | vCPU | RAM (GiB) | Current Price |
+|------------------|------|------|-----------|---------------|
+| H100 GPU         | 4    | 256  | 1,536     | $24.16        |
+| A100sxm4 GPU     | 4    | 128  | 1,024     | $8.312        |
+| FC830            | 0    | 176  | 2,048     | $3.97         |
+| FC430            | 0    | 40   | 128       | $0.75         |
+
+!!! info "How to request Bare Metal Machines?"
+
+    Interested parties who wish to explore this option are encouraged to send
+    detailed requests, including:
+
+    - Intended Use Case
+
+    - Expected Workload
+
+    - Preferred Hardware Specifications
```

Comment:
this should specify 1 of the 4 options, we don't want them making stuff up we don't have.

Reply:
@joachimweyl: Can you explain a bit more what we should specify to the general users to provide us in order to review.

Reply:
We only have the table above worth of Hardware specifications to offer so we don't want to ask them to provide a list of requests we can't meet right?